buildrunner/__init__.py

```diff
@@ -573,85 +574,92 @@ def run(self):  # pylint: disable=too-many-statements,too-many-branches,too-many
 
         exit_explanation = None
         try:  # pylint: disable=too-many-nested-blocks
+            with MultiplatformImageBuilder(keep_images=not self.cleanup_images) as multi_platform:
```

Comment:
Does this always end up starting the local registry, or just when multiple platforms are specified?

Reply:
Good catch! This is a bad design, since this will create a local registry every time. I'll need to make an update to parse the config to see if there are any multi-platform images in the platform.

Reply:
Added new commit to fix this.